test/unit/lib/plugins/aws/package/compile/events/cloudFront.test.js

```diff
@@ -1996,6 +1997,39 @@ describe('test/unit/lib/plugins/aws/package/compile/events/cloudFront.test.js',
                 },
               ],
             },
+            fnCachePolicyManagedAlternativeWay: {
```

Comment:
Maybe let's use more descriptive name instead of using suffix `AlternativeWay`? e.g. `fnCachePolicyManagedSetViaBehavior`